lib/types/src/schema/heroku.ts

```diff
@@ -1,12 +1,47 @@
-import { SchemaOutput } from '../schema'
+import { SchemaOutput, SchemaPromptGenerator } from '../schema'
+
+export interface HerokuScaling {
+  [app: string]: { [appType: string]: { size: string; quantity: number } }
+}
+
+const scaling: SchemaPromptGenerator<HerokuScaling> = async (prompt, onCancel) => {
+  console.log('You must configure the scaling for each of the Heroku apps in your pipeline.')
+  const scaling: HerokuScaling = {}
+  let allAppsConfigured = false
+  while (!allAppsConfigured) {
+    const { app } = await prompt(
+      {
+        name: 'app',
+        type: 'text',
+        message: 'Enter the name of the Heroku app to configure'
+      },
+      { onCancel }
+    )
+    const { appType, size, quantity, moreApps } = await prompt(
+      [
+        {
+          name: 'appType',
+          type: 'text',
+          initial: 'web',
+          message: `What type of app is ${app}?`
```

Comment:
**suggestion**: perhaps we can change this message to, `What process type is your app?`